Gemfile

```diff
@@ -2,6 +2,8 @@ source 'https://rubygems.org'
 
 gem 'bootstrap', '~> 4.1.3'
 gem 'config'
+gem 'devise'
+gem 'devise-remote-user'
```

Comment:
could we get a comment here - like "provides shib shim for devise" or whatever?

Reply:
"Used the same way here as all our other projects"

Reply:
The point of such comments is to give a little context for folks (like me) who are less familiar.  It is not a big ask, and it can be a real help for n00bs.  Once you know, you know ... but think about the folks that don't already know.

Reply:
I didn't know much rubygems stuff before I started working here, so I get that.  But now I find those comments distracting and frequently outdated or misleading.  `bundle info devise` is more useful and current.

Reply:
i sorta like this kind of gem comment, but i don't feel strongly about having them one way or the other.  i'm neutral on this particular one.  in this case, i doubt the comment will become outdated (it's unlikely that the purpose of the devise inclusion will change).

i think the more important Gemfile comments are of the `'we pinned to this version because...'` sort.  it can be a pain to look up what a particular gem does 5 different times when glancing at an unfamiliar project, but that is at least easily searchable online.  whereas the reasons for pinning are not necessarily easy to discern without a lot of archaeology.